integration_tests/CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ macro(RUN)
 
             add_test(${name} python ${CMAKE_CURRENT_BINARY_DIR}/${name}.py)
             set_tests_properties(${name} PROPERTIES
-                ENVIRONMENT "PYTHONPATH=${CMAKE_SOURCE_DIR}/../src/runtime/ltypes;LPYTHON_PY_MOD_NAME=${PY_MOD};LPYTHON_PY_MOD_PATH=${CMAKE_CURRENT_BINARY_DIR}")
+                ENVIRONMENT "PYTHONPATH=${CMAKE_SOURCE_DIR}/../src/runtime/ltypes;LPYTHON_PY_MOD_NAME=${PY_MOD};LPYTHON_PY_MOD_PATH=${CMAKE_SOURCE_DIR}")
```

Comment:
I think this does not seem correct. The `PY_MOD` is the shared library that is compiled from hand written C, and so it is compiled in the binary directory, not in the source directory. The `LPYTHON_PY_MOD_PATH` must point to it, thus it should be the `${CMAKE_CURRENT_BINARY_DIR}`.